cropper/app/lib/Permissions.scala

```diff
@@ -0,0 +1,30 @@
+package lib
+
+import scala.concurrent.{ExecutionContext, Future}
+import com.gu.mediaservice.lib.auth._
+
+
+object PermissionDeniedError extends Throwable("Permission denied")
+
+object Permissions {
+  val permissionStore = new PermissionStore(Config.configBucket, Config.awsCredentials)
+
+  def validateUserWithPermissions(user: Principal, permission: PermissionType.PermissionType)(implicit ec: ExecutionContext): Future[Principal] = {
+    val failFuture = Future.failed(PermissionDeniedError)
```

Comment:
Could be hoisted outside this function to avoid creating it each time (even if successful)?